src/services/README.md

```diff
@@ -66,4 +67,152 @@ It is worth noting that HdResponsive component takes a `breakpoints` prop as wel
   <div v-else-if="matches.m">m breakpoint matched (global breakpoints still available since objects are merged)</div>
 </HdResponsive>
 </template>
+```
+
+## Date
```

Comment:
You can see formatted docs here: https://github.com/SinisaG/homeday-blocks/tree/task/date-service/src/services#date
I guess it makes it a bit easier to read. :) Feedback here welcomed, about format, info, snippets,...